### PR TITLE
Feature/19 cards

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Screenshots of Card
242586070-08372013-5a08-476c-8b16-db2c7dc54eca
242586052-9eb239c3-da0a-4732-81ec-4f4e31dc5a61
242586067-17988950-718b-4fac-a1d7-ffa5fbf5b200
242586075-b3feff91-408e-4d3e-bab8-d43446c6688a

### CODE REVIEW

**Comment by ElishaSamPeterPrabhu** [comment]
Screenshots of Card

Screenshot_1685619670

Screenshot_1685619663
Screenshot_1685619667
Screenshot_1685619657

File: DemoApp/DemoApp/Views/TMCardPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        BindingContext = this;
+    }
+
+
+    private void OnClickedCommand(object e)
+    {
+        Console.WriteLine("Execute " + e);
+    }
+
+    private void isSelected_Toggled(object sender, ToggledEventArgs e)
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/TMCardPage.xaml.cs:22]: Method name should be in PascalCase.

File: DemoApp/DemoApp/Views/TMCardPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        List<TMCard> cardlist = new List<TMCard>() { card1, card2, card3, card4, card5 };
+        foreach (TMCard card in cardlist)
+        {
+            card.IsSelected = e.Value;
+        }
+    }
+
+    private void card1_Clicked(object sender, EventArgs e)
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/TMCardPage.xaml.cs:31]: Here too.

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    public static readonly BindableProperty CommandProperty =
+           BindableProperty.Create(nameof(Command), typeof(Command), typeof(TMCard), null);
+
+    public static readonly BindableProperty CommandParameterProperty =
+        BindableProperty.Create(nameof(CommandParameter), typeof(object), typeof(TMCard), null);
+    #endregion
+
+    #region Property Changes
+
+    private static void isSelectedChanged(BindableObject bindable, object oldValue, object newValue)
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs:63]: Should be in PascalCase. Consider changing this to `SelectionChanged`

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            card._border.BackgroundColor = Colors.White;
+            card._border.Stroke = Colors.Transparent;
+        }
+    }
+
+    #endregion
+
+    public TMCard()
+    {
+        _shadow = new Shadow
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs:82]: The shadow is not even showing in Android 12 simulator that I use. And it is too big in iOS. @BAL4-R Can you check this iOS and see if the shadow and border is compliant with the figma?
Simulator Screen Shot - iPhone 14 - 2023-06-02 at 12 56 04

**Review by rthanga1 — CHANGES_REQUESTED** [review]

**Comment by rthanga1** [comment]
And the branch also have conflicts aginst develop

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs (hunk 4 above)
Comment (BAL4-R) [reply Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs:82]: Seems like shadow radius is getting rendered differently in Android, iOS and Windows. 
https://github.com/dotnet/maui/issues/8796#issuecomment-1514716353

Ill use platform specific radius for this

Comment (BAL4-R) [reply Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs:93]: @rthanga1 The shadow is not visible in simulator but its visible in real device.

**Windows**
Windows

**iOS**
ios

**Android**
Android

**Review by rthanga1 — DISMISSED** [review]

**Pull request merged**